reo/tests/test_diesel_gen_sizing.py

```diff
@@ -84,19 +84,32 @@ def test_generator_sizing_with_existing_pv(self):
         d = ModelManager.make_response(run_uuid=run_uuid)
         c = nested_to_flat(d['outputs'])
 
+        c['year_one_emissions_lb_C02'] = d['outputs']['Scenario']['Site']['year_one_emissions_lb_C02']
+        c['year_one_emissions_bau_lb_C02'] = d['outputs']['Scenario']['Site']['year_one_emissions_bau_lb_C02']
+        c['utility_year_one_emissions_lb_C02'] = d['outputs']['Scenario']['Site']['ElectricTariff']['year_one_emissions_lb_C02']
+        c['utility_year_one_emissions_bau_lb_C02'] = d['outputs']['Scenario']['Site']['ElectricTariff']['year_one_emissions_bau_lb_C02']
+        c['generator_year_one_emissions_lb_C02'] = d['outputs']['Scenario']['Site']['Generator']['year_one_emissions_lb_C02']
+        c['generator_year_one_emissions_bau_lb_C02'] = d['outputs']['Scenario']['Site']['Generator']['year_one_emissions_bau_lb_C02']
+
         d_expected = dict()
-        d_expected['lcc'] = 245232.0
-        d_expected['npv'] = -4139.0
+        d_expected['lcc'] = 242935.0
```

Comment:
Have to be cautious about test values coming from private branch: are all of these changes to expected values correct for the JuMP model? (Or are they for the mosel model from the private branch?)

Reply:
@NLaws We fixed the diesel generator fuel burn bug from the private API, but that was implemented in the CHPbeta1 branch, so this may have never been merged into the develop branch of OS. I believe this change is because of that bug fix.

Reply:
I've confirmed that these are for the JuMP model.